app/services/stripe_model_callbacks/subscription_schedule/updated_service.rb

```diff
@@ -0,0 +1,46 @@
+class StripeModelCallbacks::SubscriptionSchedule::UpdatedService < StripeModelCallbacks::BaseEventService
+  TRACKED_ACTIVITIES = {
+    "subscription_schedule.canceled": :canceled
+  }.freeze.with_indifferent_access
```

Comment:
`with_indifferent_access` returns a different object that isn't frozen.

```
[2] pry(main)> {}.freeze.with_indifferent_access.frozen?
=> false
[3] pry(main)> {}.freeze.frozen?
=> true
```

```suggestion
  }.with_indifferent_access.freeze
```